src/cdk/layout/breakpoints-observer.ts

```diff
@@ -67,14 +68,15 @@ export class BreakpointObserver implements OnDestroy {
 
   /** Registers a specific query to be listened for. */
   private _registerQuery(query: string): Query {
+    query = query.trim();
```

Comment:
Should we have this one down in `splitQueries`? That way all the string manipulation logic is in one place.